charts/rancher-cis-benchmark/0.2.0/questions.yaml

```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.6.0-alpha1
+rancher_max_version: 2.6.7
```

Comment:
```suggestion
rancher_max_version: 2.6.8
```

Reply:
Again here we should probably use 2.6.99, will confirm back on this.

Reply:
Same comment as the other PR, we should make it 2.6.99

Reply:
@prachidamle @aiyengar2 I have updated the version to 2.6.99. Can you please review it again? Thanks